config/install/field.field.media.video.field_video.yml

```diff
@@ -11,11 +11,14 @@ field_name: field_video
 entity_type: media
 bundle: video
 label: 'Video URL'
-description: ''
+description: 'Enter a YouTube/Vimeo video URL. '
```

Comment:
If it is possible we should link to the docs page for VEF. We should
somehow indicate that there are video platforms supported by default and
that plugins can be installed to add more.

On 30 July 2016 at 13:33, Vijay Nandwani <notifications@github.com> wrote:

> In config/install/field.field.media.video.field_video.yml
> <https://github.com/drupal-media/media/pull/16#discussion_r72886114>:
>
> > @@ -11,11 +11,14 @@ field_name: field_video
> >  entity_type: media
> >  bundle: video
> >  label: 'Video URL'
> > -description: ''
> > +description: 'Enter a YouTube/Vimeo video URL. '
>
> Should I add something like "Visit the module page to add other providers"?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/drupal-media/media/pull/16/files/5f098189eb66cbb62df136dc8e8f44442cd74614#r72886114>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAXAORasoED47s2aA0E3WiB--XsxfW__ks5qazaVgaJpZM4JYETL>
> .
>



-- 
Janez Urevc
Portorož, Slovenija

janez@janezurevc.name
http://www.janezurevc.name

I am a FSFE fellow & supporter. You should think about becoming one too!
Visit: http://fsfe.org/support/?slashrsm